### DISCUSSION TITLE
Welcome to eaf-pyqterminal Discussions!

### DISCUSSION THREAD

**Discussion by mumu-lhl** [mumu-lhl/eaf-pyqterminal#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by manateelazycat** [comment]
eaf-pyqterminal-refresh-ms 是用于控制终端重新绘制的时间间隔吗？

如果是的话， 建议默认值是 30 或者 17毫秒， 这样整体的界面响应程度要丝滑的多。

@mumu-lhl

**Reply by mumu-lhl** [reply]
OK dd786d6
